### ISSUE TITLE
Security alert for acorn >= 7.0.0, < 7.1.1

### ISSUE THREAD

**Issue by rhodgkins** [googleapis/cloud-debug-nodejs#869]
#### Environment details

  - OS: `macOS`
  - Node.js version: `10.19.0`
  - yarn version: `1.22.4`
  - `@google-cloud/debug-agent` version: `4.2.2`

#### Steps to reproduce

Getting the following Security alert on Github for the repo containing the `@google-cloud/debug-agent` package:

> ## Remediation
>
> Upgrade **acorn** to version **7.1.1** or later. For example:
>
> ```
> acorn@^7.1.1:
>   version "7.1.1"
> ```
> _Always verify the validity and compatibility of suggestions with your codebase._
> 
> ## Details
> 
> > ### GHSA-6chw-6frg-f759 (moderate severity)
> > **Vulnerable versions:** `>= 7.0.0, < 7.1.1`
> > **Patched version:** `7.1.1`
> >
> > Affected versions of acorn are vulnerable to Regular Expression Denial of Service.
> > A regex in the form of /[x-\ud800]/u causes the parser to enter an infinite loop.
> > The string is not valid UTF16 which usually results in it being sanitized before reaching the parser.
> > If an application processes untrusted input and passes it directly to acorn,
> > attackers may leverage the vulnerability leading to Denial of Service.
> > Making sure to follow these steps will guarantee the quickest resolution possible.

In the repo, running `yarn why acorn`:

```
yarn why v1.22.4
[1/4] 🤔  Why do we have the module "acorn"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "acorn@7.0.0"
info Reasons this module exists
   - "@google-cloud#debug-agent" depends on it
   - Hoisted from "@google-cloud#debug-agent#acorn"
info Disk size without dependencies: "1.08MB"
info Disk size with unique dependencies: "1.08MB"
info Disk size with transitive dependencies: "1.08MB"
info Number of shared dependencies: 0
=> Found "espree#acorn@7.1.0"
info This module exists because "eslint#espree" depends on it.
✨  Done in 0.42s.
```

**Comment by bcoe** [comment]
@rhodgkins this security patch is included in the range of versions we accept `^7.0.0`. If you update your lock file I believe the error should go away correct?

**Comment by rhodgkins** [comment]
@bcoe apologies I'd tried lots of things including a `yarn add @google-cloud/debug-agent@latest` thinking that would do similar things! That sorted out the error thanks!
